### ISSUE TITLE
[Boss] Seethe

### ISSUE THREAD

**Issue by JonTheRealJon** [ProbablyTez/Agony#469]
https://cdn.discordapp.com/attachments/287606929163812865/311289790051909632/SHBP.png

Seethe is a Sheol-based boss who uses the power of fire to attack Isaac. He has 400 HP.

Attacks:
_______________________________________________________________________________________________________________

- Every so often, Seethe will charge up an attack and set himself in the center of the room. Once fully charged, he will breathe a large stream of fire at the player (think the Revenant in Antibirth), making the player have to circle around him, meanwhile fires are left on the ground as an extra obstacle.

- He will spawn fires in random parts of the room and disappear. Scalding will attempt to blow up the player by teleporting on a fire close to them.

  - As his health is depleted, he will channel to more fires, that are near Isaac, causing them to explode.

- Frequently spawns a fireball that bounces around the room, or a fireball the slowly follows the player. They may only disappear if the player shoots them enough times. This attack happens more often the lower his HP.

- When he dies, he will fall to the ground and explode, leaving fires where he died.

**Comment by VortexGames** [comment]
The name seems odd since a scald in a burn from hot water.

**Comment by JonTheRealJon** [comment]
True, just went with what ever, I'm bad at names tbh. So I'll just keep this as placeholder

**Comment by ProbablyTez** [comment]
Me likey! Though maybe he visually doesnt look that fire related ?

**Comment by JonTheRealJon** [comment]
He mostly intended to show up in Sheol, so I think I'll keep his colour scheme.

**Comment by JonTheRealJon** [comment]
Fuck tapped the close and comment option lol
